server/integration_tests/test_data/detection_api_context/correlatewithasthma/chart_config.json

```diff
@@ -16,7 +16,8 @@
   "context": {},
   "debug": {},
   "entities": [
-    "geoId/32"
+    "geoId/32",
```

Comment:
this expanding to include another place is expected?

Reply:
Good find.  I've fixed this!  (The Explore frontend though will only use the first one)